pom.xml

```
@@ -19,6 +19,11 @@
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/discard-old-build-plugin</gitHubRepo>
     <jenkins.version>2.346.3</jenkins.version>
+    <spotbugs.effort>Max</spotbugs.effort>
+    <spotbugs.failOnError>true</spotbugs.failOnError>
+    <spotbugs.threshold>High</spotbugs.threshold>
```

This reduced the intensity of the checks performed by spotbugs.  That is not what is intended..  We want to increase the intensity of spotbugs checks by making the threshold low, rather than medium or high.

See the specific entry in the [tutorial page](https://www.jenkins.io/doc/developer/tutorial-improve/add-more-spotbugs-checks/#increase-spotbugs-checks).
```suggestion
    <spotbugs.threshold>Low</spotbugs.threshold>
```